<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForumInteractionForm</class>
 <widget class="QDialog" name="ForumInteractionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="otherInfoTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loginTab">
      <attribute name="title">
       <string>Login Details</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Use your forum credentials</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="userNameLE"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordLE">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="saveCredentialsCB">
         <property name="text">
          <string>Save Credentials*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>*Password will be stored on your computer in clear text</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ttt">
      <attribute name="title">
       <string>Other Information</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Aircraft Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="aircraftDescriptionTE">
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Observations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="observationsTE">
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ForumInteractionForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ForumInteractionForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
